jetty-core/jetty-session/src/main/java/org/eclipse/jetty/session/AbstractSessionDataStore.java

```diff
@@ -366,6 +372,48 @@ public void setSavePeriodSec(int savePeriodSec)
         _savePeriodSec = savePeriodSec;
     }
 
+    /**
+     * Get an ObjectOutputStream suitable to serialize SessionData objects
+     * into the provided OutputStream.
```

Comment:
You need to say that this is only used by some SessionDataStores.
Ideally it would be good if all sessionDataStores could used this?

Reply:
Not all SessionDataStores will be capable of using this, as some of the persistence technologies handle their own serialization in a way in which we don't get to intervene. Eg gcloud, hazelcast. However, I can change file, mongo and maybe infinispan SessionDataStores to use this mechanism.